src/in_mem_data_store.cpp

```diff
@@ -43,25 +42,25 @@ template <typename data_t> location_t InMemDataStore<data_t>::load(const std::st
 }
 
 #ifdef EXEC_ENV_OLS
-template <typename data_t> location_t Index<data_t>::load_impl(AlignedFileReader &reader)
+template <typename data_t> location_t InMemDataStore<data_t>::load_impl(AlignedFileReader &reader)
```

Comment:
The DLVS repo has the code to read an index file using an AlignedFileReader. Please check [that implementation ](https://msasg.visualstudio.com/Bing_and_IPG/_git/ObjectStore?path=/private/Pegasus/src/DssmSearch/randnsg/src/index.cpp)  for the load_impl function that takes an AlignedFileReader as argument.

As of now, this function is not loading the index.